### PR TITLE
[kill] Lab #2

### PR DESCRIPTION
**Pull request by smthngslv**

### CODE REVIEW

File: app_python/DOCKER.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Best Docker practises
+
+1. Use slim images, if it's possible, this will reduce you image size.
+
+2. If you use `alpine` then be careful, this can increase you build time a lot.
```
Comment (DOCtorActoAntohich) [inline app_python/DOCKER.md:5]: this dude... i still have nightmares about `alpine`...

**Review by DOCtorActoAntohich — APPROVED** [review]
gud

**Comment by justmark0** [comment]
This is so smart... 💥

**Comment by Andrew174194** [comment]
33 files... OMG

File: app_python/Makefile [hunk 2]
```diff
@@ -0,0 +1,143 @@
+###--VARIABLES--########################################################################################################
+
+AWS_ACCOUNT=553667903818
```
Comment (Andrew174194) [inline app_python/Makefile:3]: Isn't it a confidential info?

File: app_python/config/nginx/.htpasswd [hunk 3]
```diff
@@ -0,0 +1 @@
+admin:$apr1$4pAYV/Ux$jAdm3GW8A1HHiba8XCH2X.
```
Comment (Andrew174194) [inline app_python/config/nginx/.htpasswd:1]: Another expose of secret info? 🤔

**Comment by Andrew174194** [comment]
This work in the nutshell:
- Ready to AWS deployment
- Powered by Nginx
- Uses Prometheus
- Has API
- That's ONLY 2nd lab
...
And it's only for showing Moscow time

image

**Review by ntdesmond — APPROVED** [review]

**Pull request closed without merging**